app/src/main/java/com/example/lapnguyen/weather_3/data/source/MainRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.example.lapnguyen.weather_3.data.source;
+
+import com.example.lapnguyen.weather_3.data.model.Currently;
```

Comment:
ERROR: Unused import: com.example.lapnguyen.weather_3.data.model.Currently. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports